client/src/app/pages/controls/business-services/components/business-service-form.tsx

```diff
@@ -35,7 +35,7 @@ export interface FormValues {
 }
 
 export interface BusinessServiceFormProps {
-  businessService?: BusinessService;
+  businessService: BusinessService | null;
```

Comment:
This seems to go against what we have done across the app for optional props. Having the prop be undefined makes sense as it is omitted for new businessServices & doesn't need to be explicitly set.

Reply:
@ibolton336, I thought about that but this isn't an optional prop anymore as it was before.
The Form requires the `businessService` prop.
The modal state value  provides either the object value itself or null:
```ts
const businessServiceToUpdate =
    createUpdateModalState !== "create" ? createUpdateModalState : null;
```